File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Essence.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Essence {

    @SerializedName("slot")
    @Expose
    public Integer slot;
    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("rank")
    @Expose
    public Integer rank;
    @SerializedName("power")
    @Expose
    public Power power;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/Essence.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Essence(
    @SerializedName("slot")
    @Expose
    var slot: Int? = null,

    @SerializedName("id")
    @Expose
    var id: Int? = null,

    @SerializedName("rank")
    @Expose
    var rank: Int? = null,

    @SerializedName("power")
    @Expose
    var power: Power? = null
)